R/plotColTile.R

```diff
@@ -0,0 +1,206 @@
+#' Plot factor data as tiles
+#' 
+#' Relative relations of two grouping can be visualized by plotting tiles with
+#' relative sizes. \code{plotColTile} and \code{plotRowTile} can be used for 
+#' this.
+#' 
+#' @param object a
+#'   \code{\link[SummarizedExperiment:SummarizedExperiment-class]{SummarizedExperiment}}
+#'   object.
+#' 
+#' @param x String specifying the column-level metadata field to show on the x-axis.
+#'   Alternatively, an \link{AsIs} vector or data.frame, see 
+#'   \code{?\link{retrieveFeatureInfo}} or \code{?\link{retrieveCellInfo}}. Must
+#'   result in a returned \code{character} or \code{factor} vector.
+#'   
+#' @param y String specifying the column-level metadata to show on the y-axis.
+#'   Alternatively, an \link{AsIs} vector or data.frame, see 
+#'   \code{?\link{retrieveFeatureInfo}} or \code{?\link{retrieveCellInfo}}. Must
+#'   result in a returned \code{character} or \code{factor} vector.
+#'   
+#' @param factor_x_by 
+#'  
+#' @param factor_y_by 
+#' 
+#' @return 
+#' A \code{ggplot2} object or \code{plotly} object, if more than one 
+#' \code{prevalences} was defined.
+#' 
+#' @name plotColTile
+#' 
+#' @examples
+#' data(GlobalPatterns)
+#' se <- GlobalPatterns
```

Comment:
Did you leave `plotRowTile(se, x="Phylum", y="Class")` out in example on purpose?

Reply:
Yes. `plotRowTile` is less usable, since the data us usually longer. However, since it is technically possible and does not require any changes, I implemented it as well.

Reply:
I thought so